builq.go

```diff
@@ -45,8 +46,28 @@ func (b *Builder) Addf(format string, args ...any) *Builder {
 	return b
 }
 
+var missingRE = regexp.MustCompile(`%!.\(MISSING\)`)
+
 func (b *Builder) Build() (string, []any, error) {
-	return b.query.String(), b.args, b.err
+	// prioritize the errors from the query string,
+	// otherwise Build might return a wrong error.
+
+	query := b.query.String()
+	switch {
+	case missingRE.MatchString(query):
```

Comment:
Why not `strings.Contains` ?

Reply:
Just want to make sure we catch the right string, not just `(MISSING)`